app/models/post.rb

```diff
@@ -1,2 +1,17 @@
 class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :comments
+  has_many :likes
+
+  after_save :update_post_counter
+
+  def update_post_counter
+    author.update(postscounter: author.posts.count)
+  end
+
+  private
+
+  def recent_comments
+    comments.order(created_at: :desc).limit(5)
+  end
```

Comment:
- [ ] Kindly remove this function from the private access control because it will be used in other classes such as controllers.

Reply:
I have implemented the changes